### ISSUE TITLE
ConvTranspose1d in INT8 mode give wrong results when bias is True

### ISSUE THREAD

**Issue by the-final** [NVIDIA/TensorRT#1840]
## Description

Is bias added to the quantitative convtranspose1d layer?
When I set bias True，the output value of TRT model is different from that of the PyTorch model.

## Environment

**TensorRT Version**: 8.2.1.8 
**NVIDIA GPU**: V100 
**NVIDIA Driver Version**: 470.82.01 
**CUDA Version**: 11.4 
**CUDNN Version**: 8.2
**Operating System**: linux 
**Python Version**: 3.8.8 
**PyTorch Version**: 1.10.1

## Steps To Reproduce

```python
import time
import torch
import torch.nn as nn
import numpy as np
import random

import tensorrt as trt
from pytorch_quantization import nn as quant_nn
from pytorch_quantization import calib
from pytorch_quantization.tensor_quant import QuantDescriptor
from pytorch_quantization import tensor_quant

seed = 1234
np.random.seed(seed)
random.seed(seed)
torch.manual_seed(seed)
torch.cuda.manual_seed(seed)

quant_nn.TensorQuantizer.use_fb_fake_quant = True

tensor_quant.QUANT_DESC_8BIT_PER_TENSOR = QuantDescriptor(num_bits=8, calib_method='max')
tensor_quant.QUANT_DESC_8BIT_CONV1D_WEIGHT_PER_CHANNEL = QuantDescriptor(num_bits=8, axis=(0))
tensor_quant.QUANT_DESC_8BIT_CONVTRANSPOSE1D_WEIGHT_PER_CHANNEL = QuantDescriptor(num_bits=8, axis=(0))

class Net(torch.nn.Module):
    def __init__(self):
        super().__init__()

        self.transpose_conv = quant_nn.ConvTranspose1d(256, 256, 6 * 2, stride=6, padding=3, output_padding=0, bias=True, quant_desc_input=tensor_quant.QUANT_DESC_8BIT_PER_TENSOR, quant_desc_weight=tensor_quant.QUANT_DESC_8BIT_CONVTRANSPOSE1D_WEIGHT_PER_CHANNEL)
#        self.transpose_conv = torch.nn.ConvTranspose1d(256, 256, 6 * 2, stride=6, padding=3, output_padding=0, bias=True)

    def forward(self, x):
        x = self.transpose_conv(x)
        return x


def collect_stats(model, data, num_batches):
    """Feed data to the network and collect statistic"""

    # Enable calibrators
    for name, module in model.named_modules():
        if isinstance(module, quant_nn.TensorQuantizer):
            if module._calibrator is not None:
                module.disable_quant()
                module.enable_calib()
            else:
                module.disable()

    model(data)

    # Disable calibrators
    for name, module in model.named_modules():
        if isinstance(module, quant_nn.TensorQuantizer):
            if module._calibrator is not None:
                module.enable_quant()
                module.disable_calib()
            else:
                module.enable()

def compute_amax(model, **kwargs):
    # Load calib result
    for name, module in model.named_modules():
        if isinstance(module, quant_nn.TensorQuantizer):
            if module._disabled:
                continue
            if module._calibrator is not None:
                if isinstance(module._calibrator, calib.MaxCalibrator):
                    module.load_calib_amax()
                else:
                    module.load_calib_amax(**kwargs)
            print(F"{name:40}: {module}")
    # model.cuda()


def onnx_to_tensorrt(onnx_model_path, trt_model_path):
    trt_logger = trt.Logger(trt.Logger.VERBOSE)
    trt_builder = trt.Builder(trt_logger)
    trt_builder.max_batch_size = 1
    explicit_batch = 1 << int(trt.NetworkDefinitionCreationFlag.EXPLICIT_BATCH)
    trt_network = trt_builder.create_network(explicit_batch)
    trt_parser = trt.OnnxParser(trt_network, trt_logger)
    with open(onnx_model_path, "rb") as f:
        assert trt_parser.parse(f.read())
    trt_config = trt_builder.create_builder_config()
    trt_config.max_workspace_size = 2 * (1 << 30)
    trt_config.flags |= 1 << int(trt.BuilderFlag.INT8)
#    trt_config.flags |= 1 << int(trt.BuilderFlag.FP16)

    trt_profile = trt_builder.create_optimization_profile()
    trt_profile.set_shape("input", (1, 256, 100), (1, 256, 100), (1, 256, 100))
    trt_config.add_optimization_profile(trt_profile)

    trt_engine = trt_builder.build_engine(trt_network, trt_config)
    with open(trt_model_path, "wb") as f:
        f.write(trt_engine.serialize())

def load_tensorrt(trt_model_path):
    trt_logger = trt.Logger(trt.Logger.WARNING)
    trt_runtime = trt.Runtime(trt_logger)
    with open(trt_model_path, "rb") as f:
        trt_engine = trt_runtime.deserialize_cuda_engine(f.read())
    trt_context = trt_engine.create_execution_context()
    return trt_context, trt_engine

def run_tensorrt(trt_context, bindings):
#    stream = torch.cuda.Stream()
#    trt_context.execute_async_v2(bindings=bindings, stream_handle=stream.cuda_stream)
#    stream.synchronize()
    trt_context.execute_v2(bindings=bindings)
    return trt_output.cpu().numpy()

if __name__ == "__main__":

    # define model
    model = Net()
    model.eval()

    print(model)

    # 数据校准
    data = torch.tensor(np.random.rand(1, 256, 100).astype(np.float32))
    with torch.no_grad():
        collect_stats(model, data, num_batches=1)
        compute_amax(model, method="percentile", percentile=100)

    input_array = np.random.rand(1, 256, 100).astype(np.float32)
    dummy_input = torch.tensor(input_array)

    # export onnx
    torch.onnx.export(
        model, 
        dummy_input, 
        "/tmp/test.onnx", 
        verbose=True, 
        export_params=True, 
        opset_version=13, 
        do_constant_folding=True, 
        input_names=["input"], 
        output_names=["output"],
        dynamic_axes={
            "input": {0: "batch_size", 2: "length"},
            "output": {0: "batch_size", 2: "length"}
        }
    )

    # onnx to tensorrt
    onnx_to_tensorrt(onnx_model_path="/tmp/test.onnx", trt_model_path="/tmp/engine.bin")

    trt_context, trt_engine = load_tensorrt("/tmp/engine.bin")
    
    # tensorrt inference
    trt_input = torch.tensor(input_array, dtype=torch.float32, device="cuda")
    trt_output = torch.empty(tuple(trt_engine.get_binding_shape(1)), dtype=torch.float32, device="cuda")
    bindings = [trt_input.data_ptr(), trt_output.data_ptr()]

    trt_out = run_tensorrt(trt_context, bindings)
    trt_out_bias = trt_out.transpose(0, 2, 1) + model.transpose_conv.bias.detach().numpy()
    trt_out_bias = trt_out_bias.transpose(0, 2, 1)

    # pytorch inference, qdq
    pytorch_out = model(dummy_input)
    pytorch_out = pytorch_out.detach().numpy()

    print("TRT out", trt_out.shape, trt_out)
    print("pytorch out", pytorch_out.shape, pytorch_out)
    print("loss", np.mean(np.abs(trt_out - pytorch_out)))
    print("add bias loss", np.mean(np.abs(trt_out_bias - pytorch_out)))

```

**Comment by ttyio** [comment]
Thanks @the-final , we have some fix in the next 8.4 release.

**Comment by ttyio** [comment]
Closing since now the 8.4EA is available, please reopen if you still have issue, thanks!
